File: LANGUAGES/1) C++/3) BOOLEAN/23.cpp
#include <iostream>

using namespace std;

int main()
{
    bool l;
    int yuz, bir, n;
    cout << " n = ";
    cin >> n;
    yuz = n / 100;
    bir = n % 10;
    l = (yuz == bir);
    cout << l;
    return 0;
}

File: LANGUAGES/4) Php/3) BOOLEAN/23.php
<?php
$n = 121;
echo " N = " . $n . "<br>";
$yuz = intval($n / 100);
$bir = $n % 10;
$l = ($yuz == $bir);
if ($l) {
    echo "True";
} else {
    echo "False";
}

